recipe/meta.yaml

```diff
@@ -28,34 +28,33 @@ outputs:
         - $RPATH/ld64.so.1    # [s390x]
     requirements:
       build:
-        - pkg-config 
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - patch  # [unix]
+        - pkg-config
       host:
         - python
         - pip
-        - certifi >=2020.06.20
-        - freetype 2.10
-        - numpy {{ numpy }}
-        - setuptools 
+        - freetype {{ freetype }}
+        - numpy 1.23         # [py<311]
+        - numpy {{ numpy }}  # [py>=311]
         - setuptools_scm >=7
```

Comment:
This should not be needed anymore.

Reply:
`setuptools_scm` is still in the list https://github.com/matplotlib/matplotlib/blob/v3.9.2/pyproject.toml#L77 and meson checks if it's avaialble https://github.com/matplotlib/matplotlib/blob/v3.9.2/meson.build#L4

Reply:
numpy is also there https://github.com/matplotlib/matplotlib/blob/v3.9.2/pyproject.toml#L81-L89. Could you clarify what exactly not needed?

Reply:
Oh, NVM then.